apps/breethe/test/breethe/sources/eea/csv_test.exs

```diff
@@ -0,0 +1,44 @@
+defmodule Breethe.Sources.EEA.CSVTest do
+  use Breethe.DataCase
+
+  alias Breethe.Sources.EEA.CSV
+
+  @sample_valid_data """
+  network_countrycode,network_localid,network_name,network_namespace,network_timezone,pollutant,samplingpoint_localid,samplingpoint_namespace,samplingpoint_x,samplingpoint_y,coordsys,station_code,station_localid,station_name,station_namespace,value_datetime_begin,value_datetime_end,value_datetime_inserted,value_datetime_updated,value_numeric,value_validity,value_verification,station_altitude,value_unit\r
+  GB,NET-GB001A,National air network,GB.GB-ExEA.AQ,http://dd.eionet.europa.eu/vocabulary/aq/timezone/UTC+02,SO2,SPO-BG0071A_00001_100,GB.GB-ExEA.AQ,00.72,34.65975799999999,EPSG:4979,GB0071A,STA-GB0071A,London,GB.GB-ExEA.AQ,2019-11-10 23:00:00+01:00,2019-11-11 00:00:00+01:00,2019-11-11 01:29:21+01:00,+01:00,3,-1,1,27,ug/m3\r
+  """
```

Comment:
Is this what you meant?

Reply:
yes exactly – at least this is exactly the same as what you'd have in an actual CSV file.